### DISCUSSION TITLE
Error with JSONEncoder when start a Python Quart application

### DISCUSSION THREAD

**Discussion by sebajou** [pallets/quart#168] [Show and tell]
Hello

I begin a POC for use Quart through a galaxie of Nodo/Mongo microservice.  

When I start Quart without Mongo, every think is fine. But after I start to intergrated Qurat-Moto I have the following error:

AttributeError: module 'quart.json' has no attribute 'JSONEncoder'

I try several Python interpretor: Poetry with Python 3.10, Python 3.10 alone, Python 3.8 alone, I always have this error.

Thanks you for your help.

```
from quart import Quart
from quart_motor import Motor

app = Quart(__name__)                 
mongo = Motor(app, uri="mongodb://localhost:27017")

if __name__ == "__main__":
    app.run(debug=True)
```

Thanks for your help

**Comment by pgjones** [comment]
See https://github.com/marirs/quart-motor/issues/4, best to stick with Quart<0.18 till it is fixed.

**Reply by sebajou** [reply]
Thanks for you help. 
I also try to directly import json (and json.jsonENCODER) from Python standart. The app is able to run. We will see if this solution bring other problems or not.
